ztp/gitops-subscriptions/argocd/README.md

```diff
@@ -94,6 +94,129 @@ If you need to remove the ArgoCD pipeline and all generated artifacts follow thi
+1. Update and restart the application configuration
+
+### Upgrade Preparation
+Obtain the new version of the GitOps ZTP container. This container can be custom built using the ../../ztp/resource-generator build directory. Extract the argocd/deployment directory as described in [Preparation of ZTP GIT repository](#preparation-of-ztp-git-repository).
+
+Update the clusters-app.yaml and policies-app.yaml to reflect the name of your Applications and the URL, branch, and path for your GIT repository.
+
+### Remove obsolete content
+If the upgrade process results in obsolete policies, for example by moving policies from one namespace to another, the steps in this section should be taken prior to upgrade to remove those policies in an automated way.
+
+1. Remove the affected PolicyGenTemplate(s) from GIT, commit and push to the remote repository.
+  1. This needs to be done prior to stopping the existing GitOps ZTP application (below) so the existing application can remove the affected resources.
+1. Wait for the changes to synchronize through the application and the affected policies to be removed from the hub cluster.
+
+Note that removing these policies from GIT, and as a result also removing them from the hub cluster, will not affect any configuration of the managed spoke clusters. Removing a policy from the hub does not delete from the spoek cluster the CRs managed by that policy.
+
+### Stop the existing GitOps ZTP applications
+Removing the existing Applications ensures that any changes to
+existing content in the GIT repository is not rolled out until the new
+version of the tooling is in place.
+
+Using the application files from the deployment directory. Note, if you used custom names for the applications you must update the names in these files first. These commands perform a non-cascaded delete which leaves in place all generated resources.
+```
+    $ oc delete -f clusters-app.yaml
+    $ oc delete -f policies-app.yaml
+```
+
+### Install Topology Aware Lifecycle Operator
+If not already installed, the [Topology Aware Lifecycle Operator](https://github.com/openshift-kni/cluster-group-upgrades-operator#readme) must be installed on the hub cluster. Refer to the steps under [Preparation of ZTP GIT repository](#preparation-of-ztp-git-repository) to install this operator.
+
+If a newer version of TALO is desired, upgrade the operator as described in the TALO documentation.
+
+### Apply required changes
+
+#### Required changes for upgrade to 4.9
+There are no required changes when upgrading from an earlier release to 4.9. It is recommended that you review required changes for later releases and incorporate those changes where applicable.
+
+#### Required changes for upgrade to 4.10
+This section only applies when upgrading from an earlier release to 4.10. In release 4.10 two additional requirements are placed on the contents of the GIT repository. Existing content in the repository must be updated to reflect this change.
+
```

Comment:
There is an Upgrade.md describes the steps to upgrade SiteConfig/PGTs examples which is pretty similar to the steps described as below. Thinking maybe move/combine this whole "Upgrading GitOps ZTP" instruction to Upgrade.md and have a referenced link in README?